src/components/Layout/Content/BottomContent/BottomContent.jsx

```
@@ -306,6 +334,54 @@ const BottomContent = () => {
       _cartItems.length > 0 ? (
         <LayerLegend></LayerLegend>
       ) : null}
+      <div className="attributionTooltipContainer">
+        <div
+          data-tooltip-id="attribution-tooltip"
+          onMouseEnter={() => dispatch(setshowMapAttribution(true))}
+          onMouseLeave={() => dispatch(setshowMapAttribution(false))}
```

Because this immediately gets set to false, if a user hovers, then leaves while trying to click one of the links, the tooltip with links closes. I think we need a timeout before `setshowMapAttribution(false)` (possibly with some debouncing)

---

good regression catch. after moving the tooltip to be a controlled component, looks like the default behavior of show/hide of react-tooltip doesn't work. added mouseEnter, mouseLeave and timeout for both the icon and tooltip content so fix.